main/src/library/RichString.flix

```diff
@@ -130,11 +477,43 @@ mod RichString {
     pub def empty(): RichString =
         RichString(Chain.empty())
 
+    ///
+    /// Returns a record containing all color definitions for foreground and background colors.
+    ///
+    /// The returned record has two fields:
+    /// - `fg`: A `ColorPalette` record containing foreground colors
+    /// - `bg`: A `ColorPalette` record containing background colors
+    ///
+    pub def colors(): {fg = ColorPalette, bg = ColorPalette} = {
```

Comment:
Im not sure I understand why ColorPalette and `colors` is here. And what is "foreground black" and "background black"